funnel/models/rsvp.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+from . import db, TimestampMixin
+from coaster.utils import LabeledEnum
+from baseframe import __
+
+__all__ = ['RSVP', 'RSVP_ACTION', 'RSVPMixin']
+
+
+class RSVP_ACTION(LabeledEnum):
+    RSVP_Y = ('Y', {'label': "I'm going", 'category': 'success', 'order': 1})
+    RSVP_N = ('N', {'label': 'Not going', 'category': 'danger', 'order': 2})
+    RSVP_M = ('M', {'label': 'Maybe', 'category': 'default', 'order': 3})
+
+
+class RSVP(TimestampMixin, db.Model):
+    __tablename__ = 'rsvp'
+    __table_args__ = (db.UniqueConstraint('user_id', 'proposal_space_id'),)
```

Comment:
ProposalSpace followed by User, and move `__table_args__` below the column definitions.